hpe_3par_kubernetes_manager.py

```diff
@@ -25,6 +25,48 @@
 timeout = 180
 
 
+def hpe_create_service_object(yml):
+    try:
+        namespace = globals.namespace
+        resp = k8s_core_v1.create_namespaced_service(namespace=namespace, body=yml)
+        logging.getLogger().debug("Service created. Status is %s" % resp.metadata.name)
+        return resp
+    except client.rest.ApiException as e:
+        logging.getLogger().error("Exception in creating service: %s" % e)
+        raise e
+
+
+def create_service(yml):
+    obj = None
+    with open(yml) as f:
+        elements = list(yaml.safe_load_all(f))
+        for el in elements:
+            logging.getLogger().debug("======== kind :: %s " % str(el.get('kind')))
+            if str(el.get('kind')) == "Service":

```

Comment:
If this condition is false, None will be returned. Is that acceptable for you?

Reply:
Yes, if the condition is false, no service yaml was passed, and hence no service is created/no object is created.